RNTester/android/app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
 
   <uses-sdk
     android:minSdkVersion="16"
-    android:targetSdkVersion="23" />
```

Comment:
Actually, delete those, they are nott needed anyone with version code and name.

Reply:
Yeah to quote the error message I'm getting in Android Studio 3.2 canary 14

> The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file.

Reply:
I believe that they are required by buck.

Reply:
This breaks Android projects with RN components (it actually doesn't let me successfully run a debug build without that removed). Is it possible to have buck adjusted? Can it not read it from the gradle file instead?

Reply:
Unfortunately I am not familiarised with buck. I will try to investigate.